README-CN.md

```diff
@@ -34,4 +37,4 @@ $ go install ./gmchart
 
 ## 开源协议
 
-MIT
\ No newline at end of file
+MIT
```

Comment:
did not change this line! guess some invisible symbol! (do you have windows? maybe different new line?)